src/relay/op/nn/nn.cc

```diff
@@ -932,15 +932,18 @@ If the input has size k on axis 1, then both gamma and beta have shape (k,).
     .set_support_level(1)
     .add_type_rel("GroupNorm", GroupNormRel);
 
-// relay.nn.batch_matmul
+// ------------------- relay.nn.batch_matmul
```

Comment:
What's this for?

Reply:
Just add the scope comment to the front and the end of these code blocks. 😄